File: en/java-spring-demo/src/main/java/com/itangcent/springboot/demo/swagger/dto/SwaggerModel.java
package com.itangcent.springboot.demo.swagger.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

@ApiModel(description = "A swagger model example")
public class SwaggerModel {
    
    @ApiModelProperty(value = "Model ID", required = true, example = "1")
    private Long id = 1L;
    
    @ApiModelProperty(value = "Model name", required = true, example = "demo")
    private String name = "demo";
    
    public Long getId() {
        return id;
    }
    
    public void setId(Long id) {
        this.id = id;
    }
    
    public String getName() {
        return name;
    }
    
    public void setName(String name) {
        this.name = name;
    }
} 

File: en/kotlin-spring-demo/src/main/kotlin/com/itangcent/springboot/kotlin/demo/swagger/dto/SwaggerModel.kt
package com.itangcent.springboot.kotlin.demo.swagger.dto

import io.swagger.annotations.ApiModelProperty

/**
 * Swagger Model Class
 * Used to demonstrate Swagger annotation usage
 */
class SwaggerModel {

    /**
     * Field A
     * Deprecated test field
     */
    @ApiModelProperty(value = "Field A", required = true)
    @Deprecated("No longer in use")
    var a: String? = null

}